frontend/packages/kubevirt-plugin/src/constants/vm/provision-source.ts

```diff
@@ -30,27 +25,27 @@ type ProvisionSourceDetails = {
 };
 
 export class ProvisionSource extends SelectDropdownObjectEnum<string> {
-  static readonly URL = new ProvisionSource('URL', {
-    label: 'URL (adds disk)',
-    description: PROVISION_SOURCE_URL_DESC,
-    order: 1,
-  });
-  static readonly DISK = new ProvisionSource('Disk', {
-    label: 'Existing PVC (adds disk)',
-    description: PROVISION_SOURCE_DISK_DESC,
-    order: 2,
-  });
-  static readonly CONTAINER = new ProvisionSource('Container', {
-    label: 'Container',
-    description: PROVISION_SOURCE_CONTAINER_DESC,
-    order: 3,
+  static readonly UPLOAD = ProvisionSource.fromDataVolume(DataVolumeSourceType.UPLOAD, 1);
+  static readonly URL = ProvisionSource.fromDataVolume(DataVolumeSourceType.HTTP, 2);
+  static readonly DISK = ProvisionSource.fromDataVolume(DataVolumeSourceType.PVC, 3);
+  static readonly CONTAINER = ProvisionSource.fromDataVolume(DataVolumeSourceType.REGISTRY, 4);
+  static readonly CONTAINER_EPHEMERAL = new ProvisionSource('CONTAINER-EPHEMERAL', {
+    label: 'Container (ephemeral)',
   });
```

Comment:
the order needs to be fixed in these two

Reply:
done